gemfire-shared/src/main/java/com/gemstone/gemfire/internal/shared/ClientSharedUtils.java

```diff
@@ -1287,8 +1287,34 @@ public static RuntimeException newRuntimeException(String message,
     return cre.newRunTimeException(message, cause);
   }
 
+  // Convert log4j.Level to java.util.logging.Level
+  public static Level converToJavaLogLevel(org.apache.log4j.Level log4jLevel) {
```

Comment:
This was done to suppress the initial warning messages seen on console at smart connector side, when the hive client gets initialized (and it tries to modify some tables in hive metastore).